pallets/stellar-bridge/src/lib.rs

```diff
@@ -199,17 +201,19 @@ pub mod pallet {
         fn offchain_worker(_n: T::BlockNumber) {
             debug::info!("Hello from an offchain worker 👋");
 
-            let res = Self::fetch_latest_txs();
+            //let res = Self::fetch_latest_txs();mlkj
+            let res = Self::fetch_latest_claimable_balances();
```

Comment:
@morgueye4 With this change, we would only allow bridge deposits through claimable balances instead of normal payments, correct?

Shouldn't we try to allow both kinds of transactions to deposit funds? Otherwise the option to use Solar or any other wallets will not work anymore.